src/api/app/controllers/source_controller.rb

```diff
@@ -360,13 +360,13 @@ def package_command
       @spkg = Package.get_by_project_and_name(origin_project_name, origin_package_name)
     end
     # rubocop:enable Metrics/LineLength
```

Comment:
That's how one can enable rubocop rules inline. There should be an ```# rubocop:disable Metrics/LineLength``` few lines above.